Zenventory/Extensions/Publisher+Ext.swift

```diff
@@ -9,7 +9,9 @@ import Combine
 import SwiftUI
 
 extension Publisher {
-    public func tryAwaitMap<T>(_ transform: @escaping (Self.Output) async throws -> T) -> Publishers.FlatMap<Future<T, Never>, Self> {
+    public func tryAwaitMap<T>(
```

Comment:
Does this function have to be 'public' though? Please unify your approach to 'access modifiers', if you started to tag extensions then keep doing it. 

I think in your app `internal` access is enough. Correct me if I'm wrong.